### DISCUSSION TITLE
Bottomless compost bucket

### DISCUSSION THREAD

**Discussion by luckystarfsh** [oldschoolgg/oldschoolbot#5236] [Ideas]
In osrs you can put compost in a bottomless compost bucket which gives 2 you uses per compost effectively halving the compost you use for farming.
Could the bucket on the bot get a perk that halves the amount of compost used for farming as well?

**Comment by LuxrayElite** [comment]
This has been raised as a suggestion in #5087  but to add to the discussion:
Now that we have a non equitable item that uses charges it may be more likely to happen. 
It would have to store each type of compost with number of charges
Charging it with any compost would provide 2 charges instead of 1 for the respective type.

**Reply by luckystarfsh** [reply]
I do agree that would be more true to how it is in osrs however in my opinion that would make it more complicated then it needs to be (commands to charge it, code to keep track of the different compost charges, code to determine when and how many charges are used, code for how to handle when you don't have enough charges, ect.).
I suggested it the way I did because I think that way would be easy to code and user friendly (just divide the amount of compost used by 2 and round up when you have it in your bank).
To be clear, that's just my opinion. I have very little experience with coding and even less with game design so I don't know for certain whether my way is better or not.
